pkg/commands/logging/newrelic/create.go

```diff
@@ -52,6 +53,7 @@ type CreateCommand struct {
 	name              string
 	placement         string
 	responseCondition cmd.OptionalString
+	region            string
```

Comment:
Apologies for the annoying nit-pick but I've been trying to order the flags alphabetically so could we move this flag and struct field up a line so the response condition flag/field comes after it.